recipes/austin/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "austin" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/P403n1x87/austin/archive/v{{ version }}.tar.gz
+  sha256: f86804f777f2f466ddede5d530d3ca67582b2a1467d000662d81272d6e9c5639
+
+build:
+  number: 0
+  entry_points:
+    - austin-tui = austin.main:main
+    - austin-web = austin.web:main
+    - austin2speedscope = austin.format.speedscope:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - autoconf
```

Comment:
You need a make dependency

Reply:
I added `make`, but windows is failing with `Unsatisfiable dependencies for platform win-64: {'autoconf', 'automake'}`. I'm not sure what the strategy should be for windows.

For linux, it's failing with `stats.c:(.text.gettime+0x1b): undefined reference to `clock_gettime'`. Am I missing some library that needs to be explicitly listed?

Reply:
I am seeing the clock_gettime related issues too elsewhere, CC @isuruf

Reply:
> Am I missing some library that needs to be explicitly listed?

Yes, `-lrt`

Reply:
@isuruf - could you be a little more explicit in where that should go/the proper way to use it? Presumably I don't need to have it in `build.sh` when building for MacOS, because that binary looks like it's already being created properly without it? If that's correct, do I need to put a section in the `build.sh` with `if [ `uname` == Darwin ]`?

Reply:
@synapticarbors `export CFLAGS="${CFLAGS} -lrt"`

Reply:
@scopatz should that just be for Linux? In the source code, MacOS doesn't use the `clock_gettime`.

Reply:
Seems like it's getting past the undefined `clock_gettime`, but is now failing on `undefined reference to 'process_vm_readv'`

Reply:
`process_vm_readv` is only in GLIBC 2.15 and we are targetting only 2.12 for now.

Reply:
@isuruf -- is there a work around for the glibc version? Or does this just have to wait for conda-forge to update to a newer glibc? If that's the case, do you have a sense of the timeline for that?

Also, is there an example recipe somewhere that shows how to handle autotools on Windows?

Reply:
>  is there a work around for the glibc version?

No

> Or does this just have to wait for conda-forge to update to a newer glibc?

Yes

> If that's the case, do you have a sense of the timeline for that?

Soon. This weekend probably.

> Also, is there an example recipe somewhere that shows how to handle autotools on Windows?

https://github.com/conda-forge/autotools_clang_conda-feedstock/blob/master/recipe/meta.yaml#L35-L68

Reply:
@isuruf -- awesome. Thanks. I'll just wait until then and try to get this fixed up

Reply:
Just to clarify, to use `glibc >= 2.15`, should I be following the following instructions, or something else?

https://conda-forge.org/docs/maintainer/knowledge_base.html#centos7